rasa/test.py

```diff
@@ -124,6 +130,84 @@ def test_nlu(model: Optional[Text], nlu_data: Optional[Text], kwargs: Optional[D
         )
 
 
+def test_compare_nlu(
+    configs: List[Text],
+    nlu: Text,
+    output: Text,
+    runs: int,
+    exclusion_percentages: List[int],
```

Comment:
What about the command line argument `--errors`?

Reply:
At the moment this script saves all prediction errors by default (in a future update I'd like to include some qualitative analysis on them).  I don't believe there is a call for the `--errors` argument to have any effect on the comparison report